From 980fcf1a36bcb3ab26d4e82a304e205f02cb376f Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Tue, 29 Jan 2019 15:03:01 +0100
Subject: [PATCH] Do not use the shell version of python-config that was
 introduced in 3.4

Revert instead to the original python version: it has our tweaks and
outputs directories correctly.

Upstream-Status: Inappropriate [oe-specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>

---
 Makefile.pre.in | 9 +++------
 1 file changed, 3 insertions(+), 6 deletions(-)

diff --git a/Makefile.pre.in b/Makefile.pre.in
index 77f91e7..5259754 100644
--- a/Makefile.pre.in
+++ b/Makefile.pre.in
@@ -1568,12 +1568,9 @@ python-config: $(srcdir)/Misc/python-config.in Misc/python-config.sh
 	sed -e "s,@EXENAME@,$(BINDIR)/python$(LDVERSION)$(EXE)," < $(srcdir)/Misc/python-config.in >python-config.py
 	@ # Replace makefile compat. variable references with shell script compat. ones; $(VAR) -> ${VAR}
 	LC_ALL=C sed -e 's,\$$(\([A-Za-z0-9_]*\)),\$$\{\1\},g' < Misc/python-config.sh >python-config
-	@ # On Darwin, always use the python version of the script, the shell
-	@ # version doesn't use the compiler customizations that are provided
-	@ # in python (_osx_support.py).
-	@if test `uname -s` = Darwin; then \
-		cp python-config.py python-config; \
-	fi
+	@  # In OpenEmbedded, always use the python version of the script, the shell
+	@  # version is broken in multiple ways, and doesn't return correct directories
+	cp python-config.py python-config
 
 
 # Install the include files
